### PR TITLE
Show compaction job progress in hsettp output

### PR DESCRIPTION
**Pull request by beckerg**
Signed-off-by: Greg Becker <gbecker@micron.com>

### CODE REVIEW

**Comment by beckerg** [comment]
```
$ while : ; do /opt/hse/bin/hsettp sched -ftab 0; echo; sleep 1; done
CNID NODE JOB ACTION   RULE Q BSY KVSETS  ALEN  CLEN PCAP COMP DGEN HBLKS KBLKS VBLKS RALEN LALEN LCLEN WMESG PROG TIME TNAME      
   1    9  35  kcomp lenmin 1   1      7 10911 10176   52    2  173     7   100   261     0     0  3101     -   78 0:57 hse_kc_ls_9
   1    8  36  kcomp lenmin 1   1      7 11032 10464   53    2  150     7   103   265     0     0  3136     -   70 0:52 hse_kc_ls_8
   1    6  37  kcomp lenmin 1   1      8 11895 10400   53    2  104     8   109   284     0     0  3387     -   51 0:40 hse_kc_ls_6
   1    4  39  kcomp lenmin 1   1      8 13007 10464   53    2   57     8   116   306     0     0  3620     -   17 0:14 hse_kc_ls_4
   1    0  40  spill rspill 0   1      5  2379  2379   12    1  353     5    25    55 -2380  2379  2379     -   51 0:12 hse_sp_sr_0
   1    2  42  kcomp lenmin 1   1      5 17879 11744   60    1  332     5    35    80     0     0  1033     -   47 0:11 hse_kc_ls_2
```

File: lib/include/hse_ikvdb/sched_sts.h [hunk 1]
```diff
@@ -74,6 +76,18 @@ sts_job_wmesg_get(const struct sts_job *job)
     return job->sj_wmesgp ? *job->sj_wmesgp : "?";
 }
 
+static inline uint
+sts_job_progress_get(const struct sts_job *job)
```
Comment (tristan957) [inline lib/include/hse_ikvdb/sched_sts.h:80]: `set_progress` and `get_progress`!

**Review by tristan957 — APPROVED** [review]

**Review by alexttx — APPROVED** [review]

**Comment by beckerg** [comment]
```
k    8   0  326    0   1.8m     0      0  24.5k  60.5m 416.1m   3211     1     2    13     1 rspill -                                                           
k    8   1  320    0   1.8m     0      0  24.5k  60.6m 416.9m   1773     1     2    13     1 rspill - 147 kvcomp,garb - 35% 0:14                                
k    8   2  314    2  22.8m     0      0  24.5k 755.9m   6.5g   1.3g     1    23   204    16 lenmin - 147 kvcomp,garb - 35% 0:14                                
k    8   3  220    4  39.6m     0      0  24.5k   1.3g   8.9g  32.6k     1    40   268     1   garb - 147 kvcomp,garb - 35% 0:14                                
n    8   4  326    4  66.2m     0      0  98.3k   2.1g  16.3g   1.3g     4    67   498    19      - 0000.0000.2ef0.e980.3030.3030.3030.3030.3030.3030
```

File: lib/include/hse_ikvdb/sched_sts.h (hunk 1 above)
Comment (tristan957) [reply lib/include/hse_ikvdb/sched_sts.h:80]: :'(

**Pull request merged**